test/e2e/subscription_e2e_test.go

```diff
@@ -464,6 +466,70 @@ func TestCreateNewSubscriptionExistingCSV(t *testing.T) {
 	compareResources(t, subscription, sameSubscription)
 }
 
+func TestSubscriptionSkipRange(t *testing.T) {
+	defer cleaner.NotifyTestComplete(t, true)
+
+	mainPackageName := genName("nginx-")
+	mainPackageStable := fmt.Sprintf("%s-stable", mainPackageName)
+	updatedPackageStable := fmt.Sprintf("%s-updated", mainPackageName)
+	stableChannel := "stable"
+	mainNamedStrategy := newNginxInstallStrategy(genName("dep-"), nil, nil)
+	mainCSV := newCSV(mainPackageStable, testNamespace, "", semver.MustParse("0.1.0-1556661347"), nil, nil, mainNamedStrategy)
+	updatedCSV := newCSV(updatedPackageStable, testNamespace, "", semver.MustParse("0.1.0-1556661832"), nil, nil, mainNamedStrategy)
+	updatedCSV.SetAnnotations(map[string]string{resolver.SkipPackageAnnotationKey: ">=0.1.0-1556661347 <0.1.0-1556661832"})
```

Comment:
Is `0.1.0-1556661832` exclusive to prevent it from being picked up as a replacement after it's installed?

Reply:
yep!